vpr/src/route/rr_graph.cpp

```diff
@@ -690,6 +690,18 @@ static void build_rr_graph(const t_graph_type graph_type,
     /* END OPIN MAP */
 
     bool Fc_clipped = false;
+    /* Draft the switches as internal data of RRGraph object
+     * These are temporary switches copied from arch switches
+     * We use them to build the edges
+     * We will reset all the switches in the function
+     *   alloc_and_load_rr_switch_inf()
+     */
+    device_ctx.rr_graph_builder.reserve_switches(device_ctx.num_arch_switches);
+    // Create the switches
+    for (int iswitch = 0; iswitch < device_ctx.num_arch_switches; ++iswitch) {
+        const t_rr_switch_inf& temp_rr_switch = create_rr_switch_from_arch_switch(iswitch, R_minW_nmos, R_minW_pmos);
+        device_ctx.rr_graph_builder.add_rr_switch(temp_rr_switch);
+    }
```

Comment:
Add a new line to the beginning and end of these codes. It will help developers to identify code blocks